Question 2.js

```diff
@@ -0,0 +1,19 @@
+function fibonacciEvenSum(total){
+    let sum = 0;
+    let num1 = 1;
+    let num2 = 2;
+    let temp = 0;
+
+    while(num1 + num2 <= total){

```

Comment:
Is the problem asking for terms whose sum is less than 4_000_000 or is it asking for terms themselves that are less than 4_000_000? The best way to figure out which one it's asking for is to change the number from 4_000_000 to 40. You can then do that with pen & paper.

Often, you're better off doing the problem with smaller inputs, then running it up to the larger inputs.